### PR TITLE
support python udf sidecar

### PR DESCRIPTION
**Pull request by joker-star-l**
**What type of PR is this?**

- [ ] API-change
- [ ] BUG
- [ ] Improvement
- [ ] Documentation
- [x] Feature
- [ ] Test and CI
- [ ] Code Refactoring

**Which issue(s) this PR fixes:**

#439

**What this PR does / why we need it:**

Not Available

**Special notes for your reviewer:**

Not Available

**Additional documentation (e.g. design docs, usage docs, etc.):**

Not Available

### CODE REVIEW

**Comment by joker-star-l** [comment]
@aylei

**Review by aylei — APPROVED** [review]
lgtm

File: api/core/v1alpha1/cnset_types.go [hunk 1]
```diff
@@ -34,6 +34,12 @@ const (
 	CNStoreStateUp       string = "Up"
 )
 
+const (
+	ContainerPythonUdf             string = "python-udf"
+	ContainerPythonUdfDefaultPort  int    = 50051
+	ContainerPythonUdfDefaultImage string = "composer000/mo-python-udf-server:latest" // TODO change it
```
Comment (aylei) [inline api/core/v1alpha1/cnset_types.go:40]: where's the dockerfile of this image?

Comment (joker-star-l) [reply api/core/v1alpha1/cnset_types.go:40]: Here: https://github.com/matrixorigin/matrixone/blob/main/optools/images/pythonserver/Dockerfile

**Pull request merged**